lib/diagnostic.js

```diff
@@ -32,6 +45,17 @@ db.dropDatabase();
 // Jon Arryn
 
 // your code begins here
+db.houses.update({
```

Comment:
Here is a solution demonstrating the uses of `$set`, `$push`, and `$push` with `$each`.

```javascript
db.houses.update({
  name: 'House Stark'
}, {
  $set: {
    members: ['Ned Stark']
  }
});

db.houses.update({
  name: 'House Targaryen'
}, {
  $set: {
    members: ['Viserys Targaryen']
  }
});

db.houses.update({
  name: 'House Arryn'
}, {
  $set: {
    members: ['Jon Arryn']
  }
});

db.houses.update({
  name: 'House Stark'
}, {
  $push: {
    members: {
      $each: ['Arysa Stark', 'Sansa Stark']
    }
  }
});

db.houses.update({
  name: 'House Targaryen'
}, {
  $push: {
    members: 'Daenerys Targaryen'
  }
});
```